galasa-managers-parent/galasa-managers-logging-parent/dev.galasa.elasticlog.manager/src/main/java/dev/galasa/elasticlog/internal/ElasticLogManagerImpl.java

```diff
@@ -50,6 +52,7 @@
 	private static final Log					logger			= LogFactory.getLog(ElasticLogManagerImpl.class);
 	public final static String					NAMESPACE		= "elasticlog";
     
+	@Reference
```

Comment:
You don't get a reference for a Manager,   the framework is passed via the initialisation method.

Reply:
it is a different mechanism fro the API